### DISCUSSION TITLE
How to reduce spawn zones ?

### DISCUSSION THREAD

**Discussion by 13RDPALEON** [Vdauphin/HeartsAndMinds#1506] [Question and answer]
Hi V, I would have liked to know how I could reduce the spawn zones when it overlaps because on some maps the zones are very large, could you help me?

![image](https://user-images.githubusercontent.com/95328067/234460779-af2045b3-47c6-4e28-a516-a49f20368445.png)

**Comment by Vdauphin** [comment]
Hello,

City overlapping is not really an issue. It only depends on how many enemies each circle will generate. With default parameter everything should be fine

You can change in mission parameters:
- reduce Density of city occupied, so you will have less enemies
- reduce Enemy density
- add Headless
- reduce Spawn city radius offset but not too much or AI will spawn too close to you
- reduce the number of group in NameLocal here https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/city/activate.sqf#L103

You can easily check effect by using ACE interaction and select Enable debug graph

Cheers

**Reply by 13RDPALEON** [reply]
So there’s no way to narrow it down?
